bindings/python/iota_sdk/client/_utils.py

```diff
@@ -52,6 +52,14 @@ def account_id_to_bech32(self, account_id: HexStr, bech32_hrp: str) -> str:
             'bech32Hrp': bech32_hrp
         })
 
+    def anchor_id_to_bech32(self, anchor_id: HexStr, bech32_hrp: str) -> str:
```

Comment:
bech32_hrp should be optional here

Reply:
I just copy-pasted, want me to change the others as well?

Reply:
No, utils needs it provided, in client it's optional

Reply:
I meant the other methods in this file, like `nft_id_to_bech32`

Reply:
Ah yes they should also have the bech32_hrp optional, otherwise there is no point in having it on the client

Reply:
Done